src/lightning_lite/accelerators/cuda.py

```diff
@@ -20,7 +20,7 @@
 import torch
 
 from lightning_lite.accelerators.accelerator import Accelerator
-from lightning_lite.utilities.imports import _TORCH_GREATER_EQUAL_1_13, _TORCH_GREATER_EQUAL_1_14
+from lightning_lite.utilities.imports import _TORCH_GREATER_EQUAL_1_13, _TORCH_GREATER_EQUAL_2_0
```

Comment:
I would still keep this...

Reply:
There's no 1.14: https://github.com/pytorch/pytorch/pull/90491

Reply:
it was nor merged, so not sure :confused:

Reply:
It is merged. See the label in the PR